dipy/viz/tests/test_fvtk.py

```diff
@@ -109,14 +109,17 @@ def test_colormap():
 @npt.dec.skipif(not fvtk.have_matplotlib)
 def test_colormaps_matplotlib():
     v = np.random.random(1000)
-    for name in 'jet', 'Blues', 'Accent', 'bone':
+    for name in 'jet', 'Blues', 'bone':
```

Comment:
On further thought, I think we should deprecate this colormap, regardless of the version of MPL that the user happens to have installed. The reasons they have for changing it into a linear segmented color map apply to us as well, and we shouldn't be using it as a continuous color map at all.